### PR TITLE
map 001-1 fix

### PR DESCRIPTION
**Pull request by HoraK-FDF**

### CODE REVIEW

File: graphics/sprites/monsters/terranite-mask.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
       <frame index="1" offsetX="-1" offsetY="-40"/>
     </animation>
     <animation direction="right">
-      <frame index="3" offsetX="-1" offsetY="-40"/>
+      <frame index="3" offsetX="1" offsetY="-40"/>
```
Comment (pazkero) [inline graphics/sprites/monsters/terranite-mask.xml:15]: I cannot merge if this adds unrelated stuff O.o

Comment (HoraK-FDF) [reply graphics/sprites/monsters/terranite-mask.xml:15]: idk what to make if i make a new pull request its always in its even a different branch.
